package/kedro_viz/data_access/managers.py

```diff
@@ -199,6 +199,9 @@ def add_pipeline(self, registered_pipeline_id: str, pipeline: KedroPipeline):
                     modular_pipelines.add_output(
                         current_modular_pipeline_id, output_node
                     )
+            # add tags
```

Comment:
we can remove this comment. as the code is obvious.

Reply:
Done.